### DISCUSSION TITLE
Can i be able to change the order flow to something else

### DISCUSSION THREAD

**Discussion by lakshmajee** [vendurehq/vendure#1559] [General]
I have a backend application (POC) recently built on top of vendure framework.
Can I change order flow status changes based on some third-party API integrations?

Is vendure framework extensible in that way ? like extending product service resolvers, orders service resolvers etc

**Comment by michaelbromley** [comment]
Hi, yes this is indeed supported. Some docs:

- https://www.vendure.io/docs/developer-guide/customizing-the-order-process/
- https://www.vendure.io/docs/plugins/plugin-examples/extending-graphql-api/

**Reply by lakshmajee** [reply]
Thanks, @michaelbromley 👍 will check them out.
